CHANGELOG.md

```diff
@@ -26,6 +26,8 @@
 N/A
 
 ## Changes since v7.4.0
+
+- [#1922](https://github.com/oauth2-proxy/oauth2-proxy/pull/1922]) Added support for env variables in the alpha struct
```

Comment:
Please do a backmerge from master and move this to the `Changes since v7.5.0` section

Reply:
done.